docs/src/debug.md

```diff
@@ -251,7 +251,7 @@ prevent the Playwright script from executing any further.
 
 Playwright runs browsers in headless mode by default. To change this behavior,
 use `headless: false` as a launch option. You can also use the [`option: slowMo`] option
-to slow down execution and follow along while debugging.
+to slow down execution (by N milliseconds per operation) and follow along while debugging.
```

Comment:
That's not entirely accurate since the notion of the 'operation' here is blurred. I.e. click is not a single operation per current implementation, which should probably be fixed.

Reply:
Interesting, I was just following https://playwright.dev/docs/api/class-browsertype#browser-type-connect which says

> Slows down Playwright operations by the specified amount of milliseconds

Is there a more correct text which I could use here in the mean time?

Reply:
That's fine, I changed the implementation to match your description.

Reply:
Ah cool, thanks!